### PR TITLE
feat(ontologies endpoint): check if entity can be deleted (DSP-1673)

### PR DESCRIPTION
**Pull request by tobiasschweizer**
resolves DSP-1673

### CODE REVIEW

File: src/api/v2/ontology/ontologies-endpoint.spec.ts [hunk 1]
```diff
@@ -904,6 +960,33 @@ describe("OntologiesEndpoint", () => {
...
+                    expect(res.canDo).toBeTrue();
+                    done();
+                }
+            );
+
+            const request = jasmine.Ajax.requests.mostRecent();
+
+            const canDeleteResourceClass = require("../../../../test/data/api/v2/ontologies/can-do-response.json");
+
+            request.respondWith(MockAjaxCall.mockResponse(JSON.stringify(canDeleteResourceClass)));
```
Comment (kilchenmann) [inline src/api/v2/ontology/ontologies-endpoint.spec.ts:980]: this constant should be renamed to `canDeleteResourcePropertyResponse`

File: src/api/v2/ontology/ontologies-endpoint.spec.ts [hunk 2]
```diff
@@ -727,6 +756,33 @@ describe("OntologiesEndpoint", () => {
...
```
Comment (kilchenmann) [inline src/api/v2/ontology/ontologies-endpoint.spec.ts:776]: To keep it consistent, this constant should be renamed to `canDeleteResourceClassResponse`

File: src/api/v2/ontology/ontologies-endpoint.spec.ts [hunk 3]
```diff
@@ -1008,6 +1091,33 @@ describe("OntologiesEndpoint", () => {
...
+                    expect(res.canDo).toBeTrue();
+                    done();
+                }
+            );
+
+            const request = jasmine.Ajax.requests.mostRecent();
+
+            const canDeleteOntoResponse = require("../../../../test/data/api/v2/ontologies/can-do-response.json");
+
+            request.respondWith(MockAjaxCall.mockResponse(JSON.stringify(canDeleteOntoResponse)));
```
Comment (kilchenmann) [inline src/api/v2/ontology/ontologies-endpoint.spec.ts:1111]: this constant should be renamed to `canReplaceCardinalityOfResourceClassResponse`

File: src/api/v2/ontology/ontologies-endpoint.spec.ts (hunk 1 above)
Comment (tobiasschweizer) [reply src/api/v2/ontology/ontologies-endpoint.spec.ts]: done in https://github.com/dasch-swiss/dsp-js-lib/pull/348/commits/c94eaaa10adfb5c680f5563e5b23bafb19dfa978

File: src/api/v2/ontology/ontologies-endpoint.spec.ts (hunk 2 above)
Comment (tobiasschweizer) [reply src/api/v2/ontology/ontologies-endpoint.spec.ts]: done in https://github.com/dasch-swiss/dsp-js-lib/pull/348/commits/c94eaaa10adfb5c680f5563e5b23bafb19dfa978

File: src/api/v2/ontology/ontologies-endpoint.spec.ts (hunk 3 above)
Comment (tobiasschweizer) [reply src/api/v2/ontology/ontologies-endpoint.spec.ts]: done in https://github.com/dasch-swiss/dsp-js-lib/pull/348/commits/c94eaaa10adfb5c680f5563e5b23bafb19dfa978

**Review by kilchenmann — APPROVED** [review]

**Pull request merged**